lib/ttfunk/subset/base.rb

```diff
@@ -106,8 +106,9 @@ def encode(options = {})
 
         tables.delete_if { |_tag, table| table.nil? }
 
-        search_range = (Math.log(tables.length) / Math.log(2)).to_i * 16
-        entry_selector = (Math.log(search_range) / Math.log(2)).to_i
+        max_pow2 = (Math.log(tables.length) / Math.log(2)).to_i
+        search_range = 2**max_pow2 * 16
```

Comment:
I wonder, doesn't RuboCop complain about spaces around power operator?

Reply:
In default configuration RuboCop actually complains if you use spaces around the power operator. I had spaces initially, but then removed them when RuboCop complained about them.